docs: 132 improve docs (#140)
Tresjs/rapier: docs/components/rigid-body.md

# RigidBody

The real-time simulation of rigid-bodies subjected to forces and contacts is the
main feature of a physics engine for video-games, robotics, or animation.

`@tresjs/rapier` provides a `RigidBody` component compatible with the `Tresjs`
ecosystem, with the advantage of making the "bound" between the two worlds
(physic world and our 3D scene).

:::info
The information in this page is a summary of the RigidBody instance,
please check the
[complete documentation](https://rapier.rs/docs/user_guides/javascript/rigid_bodies)
:::

## Basic usage

> To use a `RigidBody` component, the best case is to import it from

Reviewer: ```suggestion
To use a `RigidBody` component, import it from
```

> `@tresjs/rapier` and then pass as

Reviewer: ```suggestion
`@tresjs/rapier` and then pass as the element you want tot attach via
```

> [slot](https://vuejs.org/guide/components/slots.html#scoped-slots) the element

Reviewer: ```suggestion
[slot](https://vuejs.org/guide/components/slots.html#scoped-slots).
```

> that you want to attach.

Reviewer: ```suggestion
```

```vue
<template>
  <RigidBody>
    <TresMesh :position="[0, 8, 0]">
      <TresTorusGeometry />
      <TresMeshNormalMaterial />
    </TresMesh>
  </RigidBody>
</template>
```

## Types

> We can specify what kind of `RigidBody` type. `Dynamic` is the default.

Reviewer: ```suggestion
We can specify which kind of `RigidBody` type we want. By default is set to `Dynamic`(see below).
```

A basic floor example with type fixed:

```vue
<template>
  <RigidBody type="fixed">
    <TresMesh :position="[0, 0, 0]">
      <TresPlaneGeometry :args="[20, 20, 20]" :rotate-x="-Math.PI / 2" />
      <TresMeshBasicMaterial color="#f4f4f4" />
    </TresMesh>
  </RigidBody>
</template>
```

### Available types

> | Prop                     | Description                                                                                                  |

Reviewer: Prop?

| **restitution**          | The collider density. If non-zero the collider's mass and angular inertia will be added. (automatic-collider) | `1`                            |
| **activeCollision**      | To set the collider receiver/emitter collision events (automatic-collider)                                    | `false`                        |
| **activeCollisionTypes** | Type of the collision event. (automatic-collider)                                                             | `ActiveCollisionTypes.DEFAULT` |
| **collisionGroups**      | To specify collision groups. (automatic-collider)                                                             | `undefined`                    |

:::info
The `rigidBody` instance has many other functions, please check the
[official docs](https://rapier.rs/docs/api/javascript/JavaScript3D/) for a
complete list, if you need them, you can
use[Template ref](https://vuejs.org/guide/essentials/template-refs.html#template-refs).
:::

## Expose object

```js
const exposeObject = {
  instance: rigidBodyInstance,
  rigidBodyDesc,

>   context: colliderInfos,

Reviewer 2: Now I'm thinking to rename it `colliders` instead of `context`

Reviewer: Agree
